client/src/interface/Message.ts

```diff
@@ -18,3 +21,10 @@ export interface NewMessage {
   message?: Message;
   error?: { message: string };
 }
+
+export interface MessagingData {
+  conversation: Conversation;
+  participant: User | null;
```

Comment:
Why are we using null here and below? :)

Reply:
We are pre-populating messaging data object when fetching for data in messaging context. To set the initial values null and [] is assigned. It is required to return a messaging data object no matter outcome of the promise.